webfonts/Uno Fluent Icons/Fonts.css

```
@@ -6,7 +6,7 @@
nfbCibznHiU1be5o7cIIxvuevWZTyITt9YoDqwH297ESWKWCEE9fvUfrXWUR8i6VkxDsrsfLqtpJ0p+FJVbk3K7z/HkUJ9i2v8zHMqAtifD+gl/wWQPUE0BETSFtQKJKKEZ+aZERuOB3GQ3+hyhuQKAJfG8HMXf0OwLX9qSyxkWDYkNiuhMCWHDOLqxvKyyKlcbXUDNhuo3TXdFfkeulFQ5wxpNceGoxcdZflFMNlDodExbWkp++c/qi2Lx+t2K98IQsS1NZsGesZSgv6X7UqGU5qivOiczh31cnkjZLs+qQ0q+ipgvnmSiU57/xE85j2DB1RFTAlC7D17bIB079Wo+Wtbr37YXjePmVaSmxwf+NA69YV9wtOqLeUljEQyl/pxTncgrWyAOoXr3cNxwXvaZ1c1Tq9WpK3d2xw7O2naKpe6pLZKI4msH+4NVUp3+dibiuBcrby8pDDtpmLSBgoKjiXwwI4vY2Muf4vRjcaBoXiAuB1q6N/OinmEMUf1kQCbMloWIZutAADldf8xuaxvFpRE1MS5j2Yh+HczwD17BM4+KkneHzmmx6nrje9G+x3bjR/9bWd1sjYte2V9zzBkzNIAYj7Qfmn81e002p17M68p6G7vXX3npG9/qbDq452vsqZaQt3Txt/7Z0ZLPT7iPfcxxFSd93yjl1Zd5fOqJghppECj7sjtzz3CZA47eP+pLi1xX00e17iEIkcCZvV77HTgNWZ5P1fHx2Inw8SvcjubhhwGK/uI8/pxqi4ju9FoRd2hMP5+OVAtNhMZ09PITeLE88/o/78677CoqTczfxq/A53cQ9q4fPtw8dkbYXdKdlWwAz+SWLpRbfoljzo5jPhCmT57GBFbmKAT9dnBPjhaSmFs5HlIV0hdIdk+rsi/ABEo+w/dx4oOmHOaSjekpz84XyiUtS+SgUDhfWfB9GcNOA9un/XsCR+ub3JnuizzJHNXLbHNyjz2rDtSdPrkGe3WTbEBlJciTSuZW+JK+hpiJV7cFk9kyIRcNVIpyf4jcYTVqfnmwdSQ83X2KtgF6TBZoNNFwAACXM6fHV06PreHbBN+mSHfTXJNxD0oUKkKl2862VYAjeTW2KEgwESrBfnvbMu27v3lx2Wbx9stZRYW8oCjxH/G2CIg7JXGN+4s2XfABKgNTaxfSfswD+inufsXlCziIRCQou0QBJAHtaSHAfR8sETg5vobU5nyWnRWXdpZGbHJ1bTDvT37TnyEsXhm5P245os6G3TA66/Xg+PeyPNZOG7dgxnfe8XeN5RMVL74+qCJZ+Y6ZjvVu8xB3t9/0n4iiv0AFmY6YsNBfk+ZTV1QMeSDJsoyOd9H1zVkV96zwCv/XLJ+aRv488AiUEcnSr/OIbh0D/1BoG5bitVwsi/s2zG0W9ozvkxGHkv/n8bBx7gzOVS9VTwx1FP2gHW8DDQCX0AANVdFqkLqACciJtuuW3FD0Sd1UlSi0bqdGQSv0fmNOeppZOPNb9/okmPMbhzbo1K8eJi9YzS1kr0s6VqMckpEXnoQJsK3Vml3/ICMlseAhSdTi792/Vv19S/d/yiv4MFsgKRB7ad87oy+NCyAHF8oceumhGuY6Tl2yaabCA1OwArbWk1QWKELeFSrd+cbUuz1ujHJAyZASNuBZx2OQX2HiEwdUYvLeNzOVwlyBIkXbQgt156nqVyUP4tAKZrMhS3Gp19FmYN63g4n5Pe+uvps4JN2jpxkJfrBBq6A3iZNpSNlr0PIGCIG6KlDLhfjE3R9Z/WUFH1qsQP9SjrtAnpkOBl/FisOuJOvfXvbHQvryRff0AWvv62+O8m/jBgJt97WtPDCcnQGYjFv74vkIvoYzeiUyb0PMG1B+uVckn66SA1+6Kgt8//G7LQr38w4VC+EQ6Wq/2LTftHkz3/U1mhMdpm0i1oRdD+0fgYf1GOZyHslPPLhL6VnUwfK1IkVi+vdh9FqYcxw9axfa71V4PenZk5/UeR4TDsc7hh8gaBYiNy7wwf1cqq/dF5Hb/YajNG69EQW8YTqwvzsK35bHLLr9bvT/puSuBneMN78TdMukh+DXb+plD+VKIVcyrBZt3Z1XGzFCO78b0mLeioPVzEHI6StsRnvVXE+KKIk7UthX6uMqAnMHZlce65yrKl3fCeoNiHz8Bt3V2uaYFhxNKTi1ayAl74XRGBztR6+P3NHMj61BL83kvpPgEac8YdsVtBiZ9s57xeWVggJq0GkH0dQYcqyz7OY/tCW2aAOfaUHEeMMZneqZB9msemWpjyXsQnae/IjOnnWQDEgJg07Z1oHDRmnvB3pfZQVKyG6HVdfYq9zgAwafPZvyqN0wExWiKTZgKyOm1FlQjel5ovzxZptdfztfWItFst2hrqk2r9wPjHiRoe7Pxwt2jODOi+8rgtYzToOw4bQmvzC14cG8Q91v+yrhx+RJEAM9rdwsgueeA6oS2Imispbgfp+1WnMXTjSX4/6nuaPh9vWGGWbu2yr+//Pc98JDrOziXKOIRYlmJOSPRGQTGrhYHre5I6hXRzS8P9qnfb+82GOh6XFeighrfHOgT3R0+q+g7i+KsqtH54qMGw/v9AV1NQ8Lysg7S6fU7nw0ovSOluWeQ+edQeIdWcCudpql0N/cKrZGz7KV7/5Ifb+jj30sfsF+v9JXZ0GEOrpENxcSYsabFGHdlFYs+xozrlSZ2zvlBz/f8qXe/qNx/smMrIr4ToP+7rQO6PnVT1S/S4LpWy4nKp+NvEzdIf/VILMWh/cd6Hrb0J5xmXlh9ryebZrxMGdgv8h0f0IfzFW89bFfcC25YfyE6sSvTOTe8FBulVMP098sjLsUhD3ta2Uu/TqrYXcj4abjXGFKNPmc2seJZ+7dR9Zk6/9Ee/gkI0pq4m/8PW1vTzjFMPJIjVmMZo6jACL1nbHX1g1wK/+n5kYrXA1y+si6OOIME0P4q/UOfHE/jOKEr86Wxr448NxXsEK2DjsDO6gL7VZdnsZxlnhFcY5cf4zI5RSRF
```

```suggestion
/* https://github.com/unoplatform/uno/issues/4304 */
@font-face {
  font-family: 'Segoe UI';
  src: local('system-ui'), local('Segoe UI'), local('-apple-system'), local('BlinkMacSystemFont'), local('Inter'), local('Cantarell'), local('Ubuntu'), local('Roboto'), local('Open Sans'), local('Noto Sans'), local('Helvetica Neue'), local('sans-serif');
}
```

**Question**: Should this file include the [workaround](https://github.com/unoplatform/uno/issues/4304) we did back in 2020? I noticed with Uno.NPE the pretty critical issue it solved was _still_ present, so we might as well do apps that depend on this CSS file a favor and include it. I'll defer to your expertise, though 😄

---

@MartinZikmund I think there is only this question from @lukeblevins  left for PR ;)

---

Done, great idea @lukeblevins !